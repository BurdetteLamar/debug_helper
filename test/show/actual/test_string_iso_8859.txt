---
String (name='test_string_iso_8859' size=11 encoding=ISO-8859-1 ascii_only=true bytesize=11):
- Lorem ipsum
