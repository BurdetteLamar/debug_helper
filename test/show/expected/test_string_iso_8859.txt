---
String (message='test_string_iso_8859'):
  String#to_s: Lorem ipsum
  String#size: 11
  String#encoding: !ruby/encoding ISO-8859-1
  String#ascii_only?: true
  String#bytesize: 11
